src/uthash.h

```diff
@@ -26,9 +26,11 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 #define UTHASH_VERSION 2.0.1
 
+#ifndef uthash_noheaders
```

Comment:
This macro should be ALL_CAPS; I think I'd go with `UTHASH_NO_STANDARD_HEADERS` or else `UTHASH_FREESTANDING_IMPLEMENTATION` depending on your intended semantics.

You still `#include <stdint.h>` below, even when this macro is defined. Is that okay? Regardless, is <stdint.h> present on your platform? Regardless, is <stdint.h> guaranteed to be present on every platform, even freestanding/non-hosted implementations?